official/resnet/imagenet_main.py

```diff
@@ -165,6 +170,9 @@ def input_fn(is_training, data_dir, batch_size, num_epochs=1, num_gpus=None):
     batch_size: The number of samples per batch.
     num_epochs: The number of epochs to repeat the dataset.
     num_gpus: The number of gpus used for training.
+    dtype: Data type to use for images/features
+    datasets_num_private_threads: Number of private threads for tf.data.
```

Comment:
ditto

Reply:
Done. 
I do want to add a recommended value to the doc string as suggested by you. Perhaps we can chat about this tomorrow in person since there are a few dataset related items we need to discuss as part of this PR.